python: can't open file '/mnt/hgfs/AI': [Errno 2] No such file or directory
